package.json

```diff
@@ -105,5 +104,9 @@
       "last 1 safari version"
     ]
   },
-  "license": "GPL-3.0-or-later"
+  "license": "GPL-3.0-or-later",
+  "dependencies": {
+    "fuse-default-token-list": "^1.5.4",
```

Comment:
so you added these packages to NPM?

Reply:
yes i added those packages to npm

Reply:
got you. So after I'll publish this on `npm` https://github.com/fuseio/fuse-token-list, we can switch using this list?

Reply:
yes we can switch no problem

Reply:
please change again to `@fuseio/fuse-swap-default-token-list`  :)